frontend/src/core/store/app.js

```diff
@@ -1,10 +1,17 @@
+import requester from '../service/requester'
+
 const state = {
   isLoading: false,
-  darkMode: false
+  darkMode: false,
+  baseApiUrl: 'http://localhost:5005'
 }
 
 const actions = {
-
+  init () {
+    if (localStorage.getItem('token') != null) {
+      requester.setHeader('Authorization', localStorage.getItem('token'))
```

Comment:
In the future we will need to implement a checkToken function. 
The token stored in the localStorage might be revoked because of its expiration date.